libraries/Microsoft.Bot.Builder.AI.QnA/Utils/HttpRequestUtils.cs

```diff
@@ -70,6 +70,7 @@ public async Task<HttpResponseMessage> ExecuteHttpRequestAsync(string requestUrl
         private void SetHeaders(HttpRequestMessage request, QnAMakerEndpoint endpoint)
         {
             request.Headers.Add("Authorization", $"EndpointKey {endpoint.EndpointKey}");
+            request.Headers.Add("Ocp-Apim-Subscription-Key", $"EndpointKey {endpoint.EndpointKey}"); 
```

Comment:
in the value "EndpointKey" string should not exists for CS Key

Reply:
Modified as needed.